packages/halo/src/css/index.js

```diff
@@ -46,31 +52,31 @@ export default {
     }
   },
 
-  [`${BASE_CLASSNAME}--gap-size-default${BASE_CLASSNAME}--theme-dark`]: {
+  [`${BASE_CLASSNAME}--gap-size-default${themeClasses[themeNames.dark]}`]: {
```

Comment:
Since we have teams coding to these selector names, I believe this is a BREAKING CHANGE.